extensions/ql-vscode/test/vscode-tests/cli-integration/debugger.test.ts

```diff
@@ -0,0 +1,143 @@
+import { Selection, Uri, window, workspace } from "vscode";
+import { join } from "path";
+
+import { DatabaseManager } from "../../../src/local-databases";
+import {
+  cleanDatabases,
+  ensureTestDatabase,
+  getActivatedExtension,
+} from "../global.helper";
+import { describeWithCodeQL } from "../cli";
+import { withDebugController } from "./debug-controller";
+import { CodeQLCliServer } from "../../../src/cli";
+import { QueryOutputDir } from "../../../src/run-queries-shared";
+import { createVSCodeCommandManager } from "../../../src/common/vscode/commands";
+import { AllCommands } from "../../../src/common/commands";
+
+jest.setTimeout(30_000);
```

Comment:
This can be removed after merging in `main`.

Reply:
Done